tools/cldr-code/src/main/java/org/unicode/cldr/util/Emoji.java

```diff
@@ -368,21 +349,31 @@ public static PageId getPageId(String emoji) {
         final String major = getMajorCategory(emoji);
         final String minor = getMinorCategory(emoji);
         final PageId pageId = PageId.forString(major);
+        final Long minorOrder = minorToOrder.get(minor);
         switch (pageId) {
+            case Objects:
+                return (minorOrder < minorToOrder.get("money")) ? PageId.Objects : PageId.Objects2;
             case People:
-                if (minorToOrder.get(minor) < minorToOrder.get("person-fantasy")) {
-                    return PageId.People;
-                } else {
-                    return PageId.People2;
-                }
+                return (minorOrder < minorToOrder.get("person-fantasy"))
+                        ? PageId.People
+                        : PageId.People2;
             case Symbols:
-                if (minorToOrder.get(minor) < minorToOrder.get("transport-sign")) {
-                    return PageId.Symbols;
-                } else if (minorToOrder.get(minor) < minorToOrder.get("other-symbol")) {
-                    return PageId.Symbols2;
-                } else {
-                    return PageId.Symbols3;
-                }
+                return (minorOrder < minorToOrder.get("transport-sign"))
+                        ? PageId.Symbols
+                        : (minorOrder < minorToOrder.get("other-symbol"))
+                                ? PageId.Symbols2
+                                : PageId.Symbols3;
+            case Travel_Places:
+                return (minorOrder < minorToOrder.get("transport-ground"))
+                        ? PageId.Travel_Places
+                        : PageId.Travel_Places2;
+                // TODO: enable splitting Volume into Volume/Volume2
+                // Since they aren't emoji, the following doesn't work
+                // Reference: https://unicode-org.atlassian.net/browse/CLDR-11155
+                // case Volume:
+                //    return (minorOrder < minorToOrder.get("acre-foot"))
+                //            ? PageId.Volume
+                //            : PageId.Volume2;
```

Comment:
I realize now this whole method probably shouldn't be in the Emoji class. I need to study what corresponding mechanism can be used for non-emoji pages...

Reply:
It could go here as well, just for convenience, but I agree that for the items that are not in emoji-test.txt, we need a different mechanism. We could try categorizing by unicode General category; need to look at that.

Reply:
As it turns out, I'm keeping that method in Emoji, but only calling it if the character is in Emoji.getAllRgiNoES()

A new method PathHeader.getCharacterPageId handles the non-emoji cases (but not ₳ which is currently in limbo)